examples/ex-batched-matrix-multiply.cpp

```diff
@@ -27,299 +27,515 @@
-        Cview(i,1,0) = Aview(i,1,0)*Bview(i,0,0) + Aview(i,1,1)*Bview(i,1,0) + Aview(i,1,2)*Bview(i,2,0);
-        Cview(i,1,1) = Aview(i,1,0)*Bview(i,0,1) + Aview(i,1,1)*Bview(i,1,1) + Aview(i,1,2)*Bview(i,2,1);
-        Cview(i,1,2) = Aview(i,1,0)*Bview(i,0,2) + Aview(i,1,1)*Bview(i,1,2) + Aview(i,1,2)*Bview(i,2,2);
-        
-        Cview(i,2,0) = Aview(i,2,0)*Bview(i,0,0) + Aview(i,2,1)*Bview(i,1,0) + Aview(i,2,2)*Bview(i,2,0);
-        Cview(i,2,1) = Aview(i,2,0)*Bview(i,0,1) + Aview(i,2,1)*Bview(i,1,1) + Aview(i,2,2)*Bview(i,2,1);
-        Cview(i,2,2) = Aview(i,2,0)*Bview(i,0,2) + Aview(i,2,1)*Bview(i,1,2) + Aview(i,2,2)*Bview(i,2,2);
-
-      });
+    RAJA::forall<RAJA::omp_parallel_for_exec>(
+        RAJA::RangeSegment(0, N), [=](Index_type e) {
+
+          Cview(e, 0, 0) = Aview(e, 0, 0) * Bview(e, 0, 0)
+                           + Aview(e, 0, 1) * Bview(e, 1, 0)
+                           + Aview(e, 0, 2) * Bview(e, 2, 0);
+          Cview(e, 0, 1) = Aview(e, 0, 0) * Bview(e, 0, 1)
+                           + Aview(e, 0, 1) * Bview(e, 1, 1)
+                           + Aview(e, 0, 2) * Bview(e, 2, 1);
+          Cview(e, 0, 2) = Aview(e, 0, 0) * Bview(e, 0, 2)
+                           + Aview(e, 0, 1) * Bview(e, 1, 2)
+                           + Aview(e, 0, 2) * Bview(e, 2, 2);
+
+          Cview(e, 1, 0) = Aview(e, 1, 0) * Bview(e, 0, 0)
+                           + Aview(e, 1, 1) * Bview(e, 1, 0)
+                           + Aview(e, 1, 2) * Bview(e, 2, 0);
+          Cview(e, 1, 1) = Aview(e, 1, 0) * Bview(e, 0, 1)
+                           + Aview(e, 1, 1) * Bview(e, 1, 1)
+                           + Aview(e, 1, 2) * Bview(e, 2, 1);
+          Cview(e, 1, 2) = Aview(e, 1, 0) * Bview(e, 0, 2)
+                           + Aview(e, 1, 1) * Bview(e, 1, 2)
+                           + Aview(e, 1, 2) * Bview(e, 2, 2);
+
+          Cview(e, 2, 0) = Aview(e, 2, 0) * Bview(e, 0, 0)
+                           + Aview(e, 2, 1) * Bview(e, 1, 0)
+                           + Aview(e, 2, 2) * Bview(e, 2, 0);
+          Cview(e, 2, 1) = Aview(e, 2, 0) * Bview(e, 0, 1)
+                           + Aview(e, 2, 1) * Bview(e, 1, 1)
+                           + Aview(e, 2, 2) * Bview(e, 2, 1);
+          Cview(e, 2, 2) = Aview(e, 2, 0) * Bview(e, 0, 2)
+                           + Aview(e, 2, 1) * Bview(e, 1, 2)
+                           + Aview(e, 2, 2) * Bview(e, 2, 2);
+
+        });
     timer.stop();
-    
+
     RAJA::Timer::ElapsedType tMin = timer.elapsed();
-    if(tMin < myMin) myMin = tMin;
+    if (tMin < minRun) minRun = tMin;
     timer.reset();
   }
-  std::cout<<"Matrix Multiplication with layout 1 on CPU run time : "<<myMin<<" seconds"<<std::endl;
-//-------------------------------------------
-  
-//-------------------------------------------
-//Matrix multiply with layout 2 on the CPU
-//
-  using CPUPol2 = RAJA::loop_exec;
+  std::cout << "Performing batched matrix multiplication with layout 1 using "
```

Comment:
Please add a blank line and separator between examples here like you do for the others.

Never mind. I see what your doing now. You should print the message about which example is run BEFORE it is run in case there is an issue.